src/core/scene/a-scene.js

```diff
@@ -312,10 +397,36 @@ module.exports = registerElement('a-scene', {
 
         if (this.isPlaying) {
           this.tick(time, timeDelta);
+          this.tock(time, timeDelta);
+        }
+
+        if (this.behaviors.tickPostRender.length || this.behaviors.tockPostRender.length) {
+          // Post processing setup has been modified, emit a notification.
+          if (this.postNeedsNotify) {
```

Comment:
Sure but it only applies to post behaviors, I think doing it for ticks especially would cause lots of traffic. It's also emmited when a post behavior is removed. I don't think separate events are needed for these. The compositor will mainly listen to these in order to recompile itself when something changes in the post chain. Maybe "post-behavior-changed" or something like that?